app/assets/javascripts/modules/toggle-class.js

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
There was some misunderstanding around "making a module generic".

* making a needed behaviour easily re-usable elsewhere
* making a configurable module that would meet potential future needs (needs that may never arise), eg other event types